### DISCUSSION TITLE
How to make a readme file?

### DISCUSSION THREAD

**Discussion by Anand-Haridas7** [EddieHubCommunity/support#6490] [Q&A] [answered]
How to make a readme file?

**Comment by izam-mohammed** [comment, marked as answer]
## To create a README file on GitHub, follow these steps:

1. Navigate to the repository where you want to add the README file.
2. Click on the "Create new file" button.
3. In the "Name your file" field, enter "README.md".
4. In the editor, enter the content for your README file using Markdown syntax. Markdown is a simple formatting syntax for text on the web, and it allows you to add headings, lists, links, and another formatting to your README file.
5. When you're finished writing your README file, scroll down to the "Commit new file" section at the bottom of the page.
6. In the "Commit message" field, enter a brief message that describes the changes you've made to the README file.
7. Click the "Commit new file" button to save your changes.
